src/pyscript/__init__.py

```diff
@@ -5,7 +5,16 @@
 except ImportError:  # pragma: no cover
     import importlib_metadata as metadata  # type: ignore
 
+try:
+    import rich_click.typer as typer
+except ImportError:  # pragma: no cover
+    import typer  # type: ignore
+from rich.console import Console
+
```

Comment:
why is this `try/except` needed? In which cases `rich_click.typer` is not importable?

Reply:
This is a good call-out. I think initially I had rich integration as optional because I wasn't sure if it would work for all platforms, but looking at the dependencies in `pyproject.toml` it is not optional, so we can likely remove these import guards.